ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh

```diff
@@ -6,8 +6,11 @@ set -e
 set -u
 set -o pipefail
 
+# Canton has some issues with JDK8 so we run it with JDK11.
+JAVA="$(rlocation jdk11_nix/bin/java)"
```

Comment:
May be worth setting JAVA_HOME. I'm not sure what happens if you run a djk11 binary with JAVA_HOME pointing to jdk8. (Which might be the case here? I'm not sure who/what sets JAVA_HOME for the rest of our build.)
```
export JAVA_HOME=$(realpath $(dirname $(rlocation jdk11_nix/bin/java)/..))
```
should work I think.

Reply:
hm, I thought JAVA_HOME doesn’t matter if you invoke the binary directly but maybe I’m wrong.

Reply:
Looking through https://stackoverflow.com/questions/2025290/what-is-java-home-how-does-the-jvm-find-the-javac-path-stored-in-java-home, the JVM itself does not care about it at all. It’s used by other stuff to find java which doesn’t matter if we already invoke java.

Reply:
It's also sometimes used to find bits of java other than the executable. I'm not concerned about it not being set, as the failures modes for that are well understood (non-zero exit), but I am a little bit concerned about what happens if it is set but to a different JDK than the executable you're running.

It's likely we're not using anything that uses it, though. It's mostly used by "old" programs like Java IDEs and the like.